### DISCUSSION TITLE
customRTE bug

### DISCUSSION THREAD

**Discussion by turgunov01** [GrapesJS/grapesjs#6011] [Q&A]
Hey there!

At the moment I am doing the custom Rich Text Editor Actions as toolbar. So here is the problem, so this is my current toolbar.
<img width="694" alt="Снимок экрана 2024-07-20 в 16 31 50" src="https://github.com/user-attachments/assets/364895a8-876b-470c-a957-8e08da71141a">

So, when I edit my text, or make some actions. Some actions are doubling its html value such this:
<img width="731" alt="Снимок экрана 2024-07-20 в 16 32 56" src="https://github.com/user-attachments/assets/9f7dee82-c2ec-4f81-b5f5-f613ff5c0596">

-----------------------------------
Using Nuxt.js v3
-----------------------------------

-- Fixed

The answer was i should have empty the variable after each changing the selected element.

**Comment by mosh-tudor** [comment]
> Here is source code of the project:

But you didn't post any code 😢

**Reply by turgunov01** [reply]
oh here it is i will edit my issue question now

**Comment by artf** [comment]
Why are you re-creating actions on every `component:selected`? You should define them once

**Reply by turgunov01** [reply]
actually yes, i rewrote those moment to rte:enable, because I thought after enabling (turning on) rte editor toolbar, element is selected inside of the function and recreated every time when each unique id is selected.
